src/main/java/ui/fragments/AllBoardsFragment.java

```diff
@@ -26,4 +28,8 @@ public List<String> getAllYourBoardsTitles() {
 
         return ElementUtil.getListOfStrings(collection);
     }
+
+    public SelenideElement getSpecialBoardTitle (String boardId){
```

Comment:
```suggestion
    public SelenideElement specialBoardTitle (String boardId){
```
Pls, rename like this)